### PR TITLE
Add left nav for apps API group

### PR DESCRIPTION
**Pull request by janetkuo**
Follow up https://github.com/kubernetes/kubernetes.github.io/pull/1945#issuecomment-266874452

@devin-donnelly @kubernetes/sig-docs

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1947)

### CODE REVIEW

**Comment by janetkuo** [comment]
WIP

**Comment by janetkuo** [comment]
Done

File: docs/api-reference/README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ Use the following reference docs to understand the kubernetes REST API for vario
 * extensions/v1beta1: [operations](/docs/api-reference/extensions/v1beta1/operations.html), [model definitions](/docs/api-reference/extensions/v1beta1/definitions.html)
 * batch/v1: [operations](/docs/api-reference/batch/v1/operations.html), [model definitions](/docs/api-reference/batch/v1/definitions.html)
 * autoscaling/v1: [operations](/docs/api-reference/autoscaling/v1/operations.html), [model definitions](/docs/api-reference/autoscaling/v1/definitions.html)
+* apps/v1beta1: [operations](/docs/api-reference/apps/v1beta1/operations.html), [model definitions](/docs/api-reference/apps/v1beta1/definitions.html)
```
Comment (nikhiljindal) [inline docs/api-reference/README.md]: fwiw, this was fixed already in 1.5 branch: https://github.com/kubernetes/kubernetes.github.io/pull/1928

Comment (janetkuo) [inline docs/api-reference/README.md]: @nikhiljindal any idea why batch/v2alpha1 doesn't show me cronjobs and jobs? http://kubernetes.io/docs/api-reference/batch/v2alpha1/definitions/

Comment (janetkuo) [inline docs/api-reference/README.md]: @mbohlool any idea why batch/v2alpha1 doesn't show me cronjobs and jobs? http://kubernetes.io/docs/api-reference/batch/v2alpha1/definitions/

Comment (nikhiljindal) [inline docs/api-reference/README.md]: No idea. They are copied from core repo I think.

**Pull request merged**